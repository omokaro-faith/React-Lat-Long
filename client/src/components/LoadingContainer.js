import React from 'react';
import Loader from 'react-loader-spinner';
const LoadingContainer = (props) => (
  <div className="loader-item">
    <Loader
      type="ThreeDots"
      color="#6497b1"
      height="100"	
      width="100"
    />
  </div>
)

export default LoadingContainer;