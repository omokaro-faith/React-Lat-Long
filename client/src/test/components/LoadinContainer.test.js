import React from 'react';
import { shallow } from 'enzyme';
import LoadingContainer from '../../components/LoadingContainer';

test('should render LoadingContainer', () => {
  const wrapper = shallow(<LoadingContainer />);
  expect(wrapper).toMatchSnapshot();
});
