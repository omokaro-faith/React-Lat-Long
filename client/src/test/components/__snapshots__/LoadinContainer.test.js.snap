// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingContainer 1`] = `
<div
  className="loader-item"
>
  <Loader
    color="#6497b1"
    height="100"
    type="ThreeDots"
    width="100"
  />
</div>
`;
